crowdin_api/api_resources/abstract/resources.py

```diff
@@ -30,23 +30,39 @@ def get_page_params(
         page: Optional[int] = None,
         offset: Optional[int] = None,
         limit: Optional[int] = None,
+        labelIds: Optional[Iterable[int]] = None,
+        excludeLabelIds: Optional[Iterable[int]] = None
```

Comment:
This is wrong, the base abstract resource shouldn't be affected. The labels are not page parameters